frontend/TvCv19.Frontend/Domain/Repositories/IPatientRepository.cs

```diff
@@ -29,8 +29,7 @@ public PocPatientRepository(IPhysicianRepository physicianRepository)
 
         public Task<string> AdmitPatient(Patient patient)
         {
-            _count++;
-            patient.Id = $"{_count}";
+            patient.Id = Guid.NewGuid().ToString("N").Substring(0,20);
```

Comment:
what i found is that if you replace the hyphens it works
ie  Guid.NewGuid().ToString().Replace("-",string.empty);
just on the off chance there is identical guids at that length

Reply:
I think the ToString("N") method removes the hyphens.
Removed .Substring()